cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/task/ListTestGroupsTask.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.packerina.task;
+
+import org.ballerinalang.model.elements.PackageID;
+import org.ballerinalang.packerina.buildcontext.BuildContext;
+import org.ballerinalang.packerina.buildcontext.BuildContextField;
+import org.ballerinalang.testerina.util.TestarinaClassLoader;
+import org.ballerinalang.testerina.util.TesterinaUtils;
+import org.wso2.ballerinalang.compiler.tree.BLangPackage;
+import org.wso2.ballerinalang.compiler.tree.BLangTestablePackage;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+import static org.ballerinalang.packerina.utils.TestFileUtils.updateDependencyJarPaths;
+
+/**
+ * Task for listing groups defined in tests.
+ *
+ * @since 1.1.2
+ */
+public class ListTestGroupsTask implements Task {
```

Comment:
Do we need this task to build command. ( AFAIK groups works only with test)

Reply:
No it is only used in the test command. Is there is changes we nead to do?